resources/hpxml-measures/BuildResidentialHPXML/resources/geometry.rb

```diff
@@ -1265,8 +1265,8 @@ def self.get_wall_area_for_windows(surface, min_average_ceiling_height_for_windo
   end
 
   def self.add_windows_to_wall(surface, window_area, window_gap_y, window_gap_x, window_aspect_ratio, max_single_window_area, facade, model, runner)
-    wall_width = get_surface_length(surface)
-    average_ceiling_height = get_surface_height(surface)
+    wall_width = get_surface_length(surface) # ft
+    average_ceiling_height = get_surface_height(surface) # ft
```

Comment:
this is why we should use variable names like `wall_width_ft`!
aside: I haven't used any, but is it not common to use something like [quantiphy](https://quantiphy.readthedocs.io/en/stable/) for projects like these with lots of physical units? I guess compatibility with numpy could be an issue that could make it not worth it.

Reply:
This is ruby code. On python, I think `Pint` is the de factor standard (2k stars, versus 50 for quantiphy).

OpenStudio itself handles units, it has a quantity converter.